src/util.js

```diff
@@ -1,3 +1,5 @@
+export const root = typeof window === 'undefined' ? global : window;
```

Comment:
I was getting "window is undefined" errors. I couldn't see a convention around this in the source, though.